Parity/src/VQwBPM.cc

```diff
@@ -50,6 +50,7 @@ void VQwBPM::GetElectronicFactors(Double_t BSENfactor, Double_t AlphaX, Double_t
   Bool_t ldebug = kFALSE;
 
   fQwStriplineCalibration = BSENfactor*18.81;
+  fQwStriplineCorrection = BSENfactor*0.250014;
```

Comment:
What is BSENfactor (BPM Sensitivity... but what is it?), how does it operate in C++ since this isn't defined anywhere, and what value is it and why? Does it need to be here on either one of, or both of these two terms?

Reply:
BSENfactor is an input parameter  of the function in which you're asking this question.  This function is used while parsing the geometry file, where we can define these terms.

It serves to allow us to adjust the fQwStriplineCalibration for an individual BPM instead of the "standard" 18.81 value.  That calibration is coming from the physical construction of the BPM, and some of them are quite different.